src/main/java/ltd/clearsolutions/subtitlesparser/SubtitlesParser.java

```diff
@@ -0,0 +1,51 @@
+package ltd.clearsolutions.subtitlesparser;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SubtitlesParser {
+
+    public static List<SubtitlesData> subtitlesParser(String pathToFile) throws IOException {
```

Comment:
please remove static from method
and remove HTML from subtitles

and also you can look at this code 
https://github.com/clear-solutions/dictionary/blob/master/src/main/java/com/maxpaint/dictionary/srt/SubtitleParser.java


Wrap all code in method in trycatch 
and create custom checked exception -> SubtitlesParserException(File file, Throwable ex)
subtitlesParser(String pathToFile) -> subtitlesParser(File file)